src/js/api/config.js

```diff
@@ -92,6 +92,13 @@ const Config = function(options, persisted) {
         config.autoPause.viewability = ('viewability' in autoPause) ? !!autoPause.viewability : true;
     }
 
+    // If floating is configured to be disabled,
+    // remove the floating block from the config
+    let floating = allOptions.floating;
```

Comment:
Do we/Are we going to reassign ```floating``` at all. We can use ```const``` instead of ```let``` if thats not the case?

Also we can use ```config``` insrtead of ```allOptions``` since we've already merged ```allOptions``` and ```Defaults``` with ```config``` at this point.

Reply:
great calls! will update